client/src/pages/Login.jsx

```
@@ -71,6 +71,14 @@ export default function Login() {
           </div>
         )}
       </button>
+      <button className={styles.demoLoginButton}>
+        데모용 로그인
+        {lastProvider === 'demo' && (
+          <div className={styles.highlighted} onClick={(e) => e.stopPropagation()}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)